hack/update-generated-deep-copies.sh

```diff
@@ -15,9 +15,9 @@ os::build::setup_env
 function result_file_name() {
 	local version=$1
 	if [ "${version}" == "api" ]; then
-		echo "pkg/api/deep_copy_generated.go"
+		echo "${OUTPUT_DIR_ROOT}/deep_copy_generated.go"
```

Comment:
> sure, but it's not really important, is it?

It reflects the actual path of where we expect these things to live.  Put the `${OUTPUT_DIR_ROOT}` before `pkg/api`.